include/language.php

```diff
@@ -401,8 +401,10 @@ public function __isset( $property ) {
 	 * @return void
 	 */
 	private function deprecated_property( $property, $replacement ) {
-		/** This filter is documented in wordpress/wp-includes/functions.php */
-		if ( WP_DEBUG && apply_filters( 'deprecated_function_trigger_error', true ) ) {
+		/**
+		 * This filter doesn't exist in WordPress core yet, but let's use a similar system as `deprecated_function_trigger_error`.
+		 */
```

Comment:
```suggestion
		/**
		 * Filters whether to trigger an error for deprecated properties.
		 *
		 * The filter name is intentionnaly not prefixed to use the same as WordPress
		 * in case it is added in the future. 
		 *
		 * @since 3.4
		 *
		 * @param bool $trigger Whether to trigger the error for deprecated properties. Default true.
		 */
```